openhands/core/logger.py

```diff
@@ -157,7 +157,8 @@ def log_uncaught_exceptions(ex_cls, ex, tb):
 
 sys.excepthook = log_uncaught_exceptions
 openhands_logger = logging.getLogger('openhands')
-current_log_level = logging.INFO
+current_log_level = logging.DEBUG
+# current_log_level = logging.INFO
```

Comment:
I'm not sure we want to set the default log level to DEBUG.

Reply:
Sorry. This was pushed to the wrong repo.